build-tools/fetch-windows-assemblers/fetch-windows-assemblers.cs

```diff
@@ -0,0 +1,697 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+//
+// Taken from:
+//   https://github.com/force-net/Crc32.NET/blob/fbc1061b0cb53df2322d5aed33167a2e6335970b/Crc32.NET/SafeProxy.cs
```

Comment:
We should add a `build-tools/fetch-windows-assemblers.tpnitems` file so that we can properly track the license information and source location.